core/domain/skill_services.py

```diff
@@ -407,12 +435,12 @@ def _filter_skills_by_classroom(augmented_skill_summaries, classroom_name):
     Args:
         augmented_skill_summaries: list(AugmentedSkillSummary).
             The list of augmented skill summaries.
-        classroom_name: str. The classroom_name of the topic to which skill is
-            assigned to.
+        classroom_name: str. The classroom_name of the topic to which the skill
+            is assigned to.
 
     Returns:
         list(AugmentedSkillSummary). The list of augmented
-            skill summaries with the given classroom name.
+        skill summaries with the given classroom name.
```

Comment:
You can move some words to the previous line.

Reply:
Done